.github/workflows/php.yml

```diff
@@ -0,0 +1,24 @@
+name: PHP tests
+on: [pull_request]
+jobs:
+  php-cs-fixer:
+    name: PHP-CS-FIXER
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 0
+      - name: php-cs-fixer default
+        uses: prestashopcorp/github-action-php-cs-fixer@master
+  phpstan:
+    name: PHPSTAN
```

Comment:
Looks like GitHub is yelling at us

![image](https://user-images.githubusercontent.com/6768917/70627708-95708f80-1c1e-11ea-811e-7e8b64c08fc9.png)


```suggestion
    name: PHPStan
```